packages/types/src/discord/user.ts

```diff
@@ -40,6 +40,8 @@ export interface DiscordUser {
   avatar_decoration_data?: DiscordAvatarDecorationData
   /** data for the user's collectibles */
   collectibles?: DiscordCollectibles
```

Comment:
```suggestion
  avatar_decoration_data?: DiscordAvatarDecorationData | null
  /** data for the user's collectibles */
  collectibles?: DiscordCollectibles | null
```

<https://discord.com/developers/docs/resources/user#user-object-user-structure>